### DISCUSSION TITLE
Accuracy flag for convergence criterion with SCC iterations

### DISCUSSION THREAD

**Discussion by gavinwinter** [tblite/tblite#67] [Q&A]
Does the float specified with the `accuracy` flag in the `set()` function of the [tblite calculator here](https://tblite.readthedocs.io/en/latest/_modules/tblite/interface.html#Calculator.set) exactly correspond with the units for energy convergence found in the `--acc` flag in the [original xtb code here](https://xtb-docs.readthedocs.io/en/latest/sp.html#accuracy-and-iterations)?

Thanks!

**Comment by awvwgk** [comment]
Yes the thresholds correspond closely to the ones used in `xtb`, however I'm not exactly matching them in every detail. As in `xtb`, a lower value for the accuracy results in tighter convergence thresholds and larger integral cutoffs.
